README.md

```diff
@@ -8,11 +8,12 @@
 # Requirements
 1. `python 3.10`
 2. `git`
+3. A full install of a supported Linux distro (Live USB will not work)
```

Comment:
IMO this should more be like `NOTE: Live images will not work! You need a full install.` instead of being in requirements.